Sources/Plasma/PubUtilLib/plAudio/plEAXEffects.h

```diff
@@ -49,58 +50,46 @@ You can contact Cyan Worlds, Inc. by email legal@cyan.com
 #ifndef _plEAXEffects_h
 #define _plEAXEffects_h
 
+#include <set>
+#include <efx.h>
+#include <efx-presets.h>
```

Comment:
It would be nice to avoid pulling these library headers into a one of our headers. Are there any forward declarations we can use to avoid this?